_codux/boards/services-section.board.tsx

```diff
@@ -0,0 +1,85 @@
+import { createBoard } from '@wixc3/react-board';
+import classNames from 'classnames';
+import styles from './services-section.board.module.scss';
+
+export default createBoard({
+    name: 'ServicesSection',
+    Board: () => (
+        <div className={styles.servicesSection}>
```

Comment:
> Also because we realized we can't use CSS/SASS modules, now it's more important that all classes in the snippet have a common unique prefix.

I'm not sure if we need it. When we drop a snippet into a component with a CSS/SCSS modules, it doesn't matter. 
And in case of a plain CSS/SCSS a prefix will be autogenerated (it is relevant for built-in snippet, but I guess we will reuse it)

Reply:
How the prefix is generated? Is it based on the board file name / board "name" property?

Reply:
It is a board file name or a component name

Reply:
> When we drop a snippet into a component with a CSS/SCSS modules, it doesn't matter

If I add 'services section' and 'hero banner' snippets to my component, and if both of those snippets contain a button, then in the resulting component we'll have 'button1' and 'button2'.

Reply:
From this perspective prefixing makes sense

Reply:
But does it mean that if we add a prefix to each class name, then Codux will add another prefix and we'll up with `.servicesSectionServicesSectionTitle`?

Reply:
To summarize:

1. If the destination component uses plain CSS:
   `<div className="button">` will be transformed into `<div className="SnippetName_button">`
2. If the destination component uses CSS modules:
   `<div className="button">` will be transformed into `<div className={styles.foo}>`

Manually prefixing classes in the snippet's CSS file is bad for scenario 1, but good for scenario 2.

I think scenario 2 is more common. Our own templates use modules by default. Which means we should add manual prefixes.